src/pytest_devlife/settings.py

```diff
@@ -1,12 +1,29 @@
 from pathlib import Path
 import os
+import os.path as osp
+import json
 
+def _load_settings_from_disk():
+    try:
+        settings_file = osp.join(Path.home(), 'DevLife', '.vscode', 'settings.json')
+        with open(settings_file) as f:
+            settings = json.load(f)
+            if 'insper.token' in settings and \
+            'insper.hostname' in settings:
+                return {
+                    'token': settings['insper.token'],
+                    'hostname': settings['insper.hostname']
+                }
+    except:
+        pass
```

Comment:
This sequence is logically correct, but I think it would be better if we repeated the `raise RuntimeError()` because of exception chaining: https://docs.python.org/3/tutorial/errors.html#exception-chaining

```python
except:
    raise RuntimeError()

raise RuntimeError()
```

We may also not need the last `raise RuntimeError()` outside the `try...except` if we do this:

```python
def _load_settings_from_disk():
    try:
        settings_file = osp.join(Path.home(), 'DevLife', '.vscode', 'settings.json')
        with open(settings_file) as f:
            settings = json.load(f)
            return {
                'token': settings['insper.token'],
                'hostname': settings['insper.hostname']
            }
    except:
        raise RuntimeError()
```

I think it would work, since `settings['insper.token']` or `settings['insper.hostname']` would raise a `KeyError`.